docs/user/explanation/continuous-documentation.rst

```diff
@@ -7,28 +7,50 @@
 
 
 
-Continuous Documentation Deployment
-===================================
+Continuous Documentation
+========================
 
-Read the Docs is a *Continuous Documentation Deployment* platform for your software project.
-Every time you change something in your documentation, Read the Docs will detect your change and build your documentation.
+Read the Docs is a *Continuous Documentation* platform for your software project.
+Every time you change something in your documentation, Read the Docs will automatically detect your changes,
+*build* your documentation,
+and finally *deploy* your documentation for readers to see.
 
-The Continuous Integration and Continuous Deployment (CI/CD) features are configured with your repository provider,
-such as GitHub, Bitbucket or GitLab.
-With each change committed to your repository, we are notified by the configured *webhook*.
+The Continuous Integration and Continuous Deployment (:term:`CI/CD`) features are configured with your repository provider,
+such as |git_providers_or|.
+With each change committed to your repository, we are notified by the configured :term:`webhook <webhooks>`.
 
-When a receive a *webhook* notification, we match it to a project's *Integration*.
-When a webhook is received, the matching project will then:
+When a receive a *webhook* notification, we match it to a Read the Docs project.
+The matching project will then process your build and publish the documentation.
 
-* :doc:`Build </builds>` the latest commit.
-* Synchronize your versions based on the latest tag and branch data in Git.
-* Run your :doc:`automation rules</automation-rules>`.
-* Auto-cancel any currently running builds of the same version.
-* Add a log entry to the integration's :guilabel:`Recent Activity`.
```

Comment:
I'm a little confused how we keep migrating this page away from actually documenting the workflow for integrations. Do we have a page in the docs that explains how to setup GitHub on your repo, and what that flow looks like?

Reply:
Changes to add a feature for Git provider integrations are now parked here https://github.com/benjaoming/readthedocs.org/pull/5

I'll move them once this PR is merged.